tests/cli_test.py

```diff
@@ -761,12 +761,25 @@ def test_configure_command_create_vdir(runner):
     assert '{}/khal/calendars/private1' .format(runner.xdg_data_home) in actual_config
 
 
+def cleanup(paths):
+    """reset permissions of all files and folders in `paths` to 666 resp. 777"""
+    for path in paths:
+        if os.path.exists(path):
+            os.chmod(str(path), 0o777)
+        for dirpath, dirnames, filenames in os.walk(path):
+            os.chmod(str(dirpath), 0o777)
+            for filename in filenames:
```

Comment:
Should this loop through `dirnames` as well?

Reply:
I don't think so. As os.walk walks down all directories, we catch all directories eventually with line 770.

Reply:
Ah, yes, re-reading it I think you're right.